aws-s3-transfer-manager/src/config.rs

```diff
@@ -135,8 +135,8 @@ impl Builder {
     ///
     /// This _optional_ name is used to identify the application in the user agent that
     /// gets sent along with requests.
-    pub fn app_name(mut self, app_name: Option<AppName>) -> Self {
-        self.app_name = app_name;
+    pub fn frame_metadata(mut self, frame_metadata: Option<FrameworkMetadata>) -> Self {
```

Comment:
**suggestion**: Spell this out, `framework_metadata`.

**suggestion**: Maybe doc hide these config functions as most people shouldn't need to care or know about them.

Applies here and elsewhere